src/index.js

```diff
@@ -1,22 +1,40 @@
-import React, { useState } from "react";
-import AppNavigator from "./navigation/navigators/AppNavigator";
+import React, { useState, useEffect } from "react";
+import AsyncStorage from "@react-native-community/async-storage";
+import {
+  AppExtendedContainer,
+  AppPrimaryContainer
+} from "./navigation/navigators/AppNavigator";
 import TranslationProvider, {
   Translation
 } from "./components/TranslationProvider";
 import { changeLocale } from "./i18n";
 
-const AppContainer = () => {
+const AppLocalised = () => {
   const [locale, changeLocaleState] = useState("ru");
+  const [isFirstLaunch, toggleFirstLaunch] = useState(true);
+  let UltimateNavigator = AppExtendedContainer;
+
   const onLocaleChange = localeKey => {
     changeLocale(localeKey);
     changeLocaleState(localeKey);
   };
 
+  useEffect(() => {
+    AsyncStorage.getItem("lang").then(langSet => {
+      toggleFirstLaunch(langSet);
```

Comment:
It's a bit strange to set language to toggleFirstLaunch state prop :))
Apart from that as far as I understand isFirstLaunch prop will be true whenever user closes and launches app

Reply:
Language is set after pressing one of the buttons on the intro screen, so I thought it can be used.
Fixed the _isFirstLaunch_ prop